.travis.yml

```diff
@@ -49,7 +49,7 @@ matrix:
 
         # Try MacOS X
         - os: osx
-          env: PYTHON_VERSION=2.7 SETUP_CMD='test'
+          env: PYTHON_VERSION=2.7 SETUP_CMD='test -a '-vv''
```

Comment:
https://travis-ci.org/gammapy/gammapy/jobs/110596524#L344

Maybe try to change the quoting like this:
```
env: PYTHON_VERSION=2.7 SETUP_CMD='test -a "-vv"'
```
or this:
```
```
env: PYTHON_VERSION=2.7 SETUP_CMD="test -a '-vv'"
```
or remove this change (since just adding this to one, but not the other builds isn't too useful).